scene/gui/control.h

```diff
@@ -201,6 +201,8 @@ class Control : public CanvasItem {
 		Ref<Theme> theme;
 		Control *theme_owner = nullptr;
 		Window *theme_owner_window = nullptr;
+		StringName theme_custom_type = StringName();
```

Comment:
StringNames don't need to be initialized.

Reply:
Good catch, same with `window.h`